src/Libraries/OrchardCore.Commerce.MoneyDataType/Currency.extra.cs

```diff
@@ -161,6 +184,9 @@ public static ICurrency FromRegion(RegionInfo region, IEnumerable<ICurrencyProvi
 
     public static ICurrency FromCulture(CultureInfo culture, IEnumerable<ICurrencyProvider> providers = null)
     {
+        ArgumentNullException.ThrowIfNull(culture);
+        if (culture.TryGetRegionInfo().ISOCurrencySymbol == "EUR") return Euro;
```

Comment:
```suggestion
        if (culture.TryGetRegionInfo()?.ISOCurrencySymbol == "EUR") return Euro;
```
Possible NRE.

Reply:
I assumed that's the same case as the culture being null, but you are right. But then it's best to refactor the whole thing to handle null culture or invalid culture that doesn't return a region, and return UnspecifiedCurrency in both cases. Then this method never returns null.